controllers/reps.js

```diff
@@ -1,9 +1,12 @@
-const log = require('../models/reps');
+const logReps = require('../models/reps');
 
 module.exports = {
   index,
 };
 
 function index(req, res) {
-  res.send('reps/index')
-}; 
\ No newline at end of file
+  logReps.find({}, function (err, reps) {
+    res.render('user/index');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
    res.render("user/index");
```